samples/apps/copilot-chat-app/webapi/CopilotChat/Skills/ChatSkills/CopilotChatPlanner.cs

```diff
@@ -43,21 +47,84 @@ public CopilotChatPlanner(IKernel plannerKernel, PlannerOptions? plannerOptions)
     /// Create a plan for a goal.
     /// </summary>
     /// <param name="goal">The goal to create a plan for.</param>
+    /// <param name="availableFunctionsOnly">Optional override to allow developers to keep functions regardless of whether they're available in the planner's context or not.</param>
     /// <returns>The plan.</returns>
-    public Task<Plan> CreatePlanAsync(string goal)
+    public async Task<Plan> CreatePlanAsync(string goal, bool availableFunctionsOnly = true)
     {
         FunctionsView plannerFunctionsView = this.Kernel.Skills.GetFunctionsView(true, true);
         if (plannerFunctionsView.NativeFunctions.IsEmpty && plannerFunctionsView.SemanticFunctions.IsEmpty)
         {
             // No functions are available - return an empty plan.
-            return Task.FromResult(new Plan(goal));
+            return new Plan(goal);
         }
 
-        if (this._plannerOptions?.Type == PlanType.Sequential)
+        Plan plan = this._plannerOptions?.Type == PlanType.Sequential
+                ? await new SequentialPlanner(this.Kernel, new SequentialPlannerConfig { RelevancyThreshold = 0.75 }).CreatePlanAsync(goal)
```

Comment:
nit: 0.75 might be high depending on your scenarios, feel free to modify this up/down based on your testing. Probably a good thing to put in _plannerOptions to be able to change easily.

Reply:
Oh I meant to remove this completely. Does relevancy effect the outcome as much when using gpt-4?

Reply:
Changed planner default back to Action but added RelevancyThreshold as plannerOption with note to use 0.5 if using SequentialPlanner with gpt3.5turbo